addons/im_livechat/static/tests/embed/livechat_service.test.js

```diff
@@ -81,16 +82,26 @@ test("Only necessary requests are made when creating a new chat", async () => {
     const livechatChannelId = await loadDefaultEmbedConfig();
     const operatorPartnerId = serverState.partnerId;
     onRpcBefore((route, args) => {
-        if (!route.includes("assets") && !route.includes("lazy_session_info")) {
+        if (!route.includes("assets")) {
             asyncStep(`${route} - ${JSON.stringify(args)}`);
         }
     });
     await start({ authenticateAs: false });
     await contains(".o-livechat-LivechatButton");
     await waitForSteps([
-        `/mail/action - ${JSON.stringify({
-            fetch_params: [["init_livechat", livechatChannelId]],
-            context: { lang: "en", tz: "taht", uid: serverState.userId, allowed_company_ids: [1] },
+        `/web/dataset/call_kw/ir.http/lazy_session_info - ${JSON.stringify({
```

Comment:
Note: lazy session instead of /mail/data because mail/core/web is loaded in test bundle